packages/nextcloud/generate_props.dart

```diff
@@ -36,18 +36,18 @@ void main() {
       'lock-owner-displayname': 'String',
       'lock-owner-editor': 'String',
       'lock-owner-type': 'int',
-      'lock-time': 'int',
+      'lock-time': 'unixEpoch',
       'lock-timeout': 'int',
```

Comment:
This could be parse as a Duration

Reply:
What do you mean?

It must be a DateTime according to the upstream docs: https://github.com/nextcloud/documentation/blob/master/developer_manual/client_apis/WebDAV/basic.rst#supported-properties
> Date when the lock was created as a timestamp.

Reply:
No the timeout which is the TTL in seconds. The code preview shows too much context.